rmw_opendds_shared_cpp/include/rmw_opendds_shared_cpp/node.hpp

```diff
@@ -15,29 +15,16 @@
 #ifndef RMW_OPENDDS_SHARED_CPP__NODE_HPP_
 #define RMW_OPENDDS_SHARED_CPP__NODE_HPP_
 
-#include "rmw/types.h"
-
 #include "rmw_opendds_shared_cpp/visibility_control.h"
 
-RMW_OPENDDS_SHARED_CPP_PUBLIC
-rmw_node_t *
-create_node(
-  const char * implementation_identifier,
-  const char * name,
-  const char * namespace_,
-  size_t domain_id);
-
-RMW_OPENDDS_SHARED_CPP_PUBLIC
-rmw_ret_t
-destroy_node(const char * implementation_identifier, rmw_node_t * node);
+#include "rmw/types.h"
 
 RMW_OPENDDS_SHARED_CPP_PUBLIC
-const rmw_guard_condition_t *
-node_get_graph_guard_condition(const rmw_node_t * node);
+rmw_node_t *
+create_node(rmw_context_t & context, const char * name, const char * namespace_);
```

Comment:
```suggestion
create_node(rmw_context_t & context, const char * name, const char * namespace);
```

Reply:
I don't like namespace_ as a parameter either, since underscore suffix is usually used for member variables. So I changed it to name_space in the .cpp file, but for got it here in .hpp. I understand the original namespace_ is to avoid collision with the cpp keyword namespace.

Reply:
Yeah, `name_space` is better, I forgot `namespace` is a keyword.